### PR TITLE
py-cinemasci: new version 1.7.0

### PR DESCRIPTION
**Pull request by EthanS94**
This PR adds the newest version of `py-cinemasci`

### CODE REVIEW

**Review by eugeneswalker — DISMISSED** [review]
Checksum verified. Thanks!

File: var/spack/repos/builtin/packages/py-cinemasci/package.py [hunk 1]
```diff
@@ -11,12 +11,13 @@ class PyCinemasci(PythonPackage):
...
 
     homepage = "https://github.com/cinemascience"
-    pypi = "cinemasci/cinemasci-1.3.tar.gz"
+    pypi = "cinemasci/cinemasci-1.7.0.tar.gz"
 
     tags = ['e4s']
 
     maintainers = ['EthanS94']
 
+    version('1.7.0', sha256='70e1fa494bcbefdbd9e8859cdf1b01163a94ecffcdfa3da1011e4ef2fcee6169', preferred=True)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-cinemasci/package.py]:
```suggestion
    version('1.7.0', sha256='70e1fa494bcbefdbd9e8859cdf1b01163a94ecffcdfa3da1011e4ef2fcee6169')
```
Spack always prefers the latest stable release by default, no need to specify this

File: var/spack/repos/builtin/packages/py-cinemasci/package.py [hunk 2]
```diff
@@ -29,5 +30,8 @@ class PyCinemasci(PythonPackage):
     depends_on('py-h5py+mpi', when='+mpi', type=('build', 'run'))
     depends_on('py-ipywidgets', type=('build', 'run'))
     depends_on('py-jupyterlab', type=('build', 'run'))
+    depends_on('py-matplotlib', type=('build', 'run'), when='@1.7.0:')
     depends_on('py-pandas', type=('build', 'run'))
+    depends_on('py-scikit-image', type=('build', 'run'), when='@1.7.0:')
+    depends_on('py-scipy', type=('build', 'run'), when='@1.7.0:')
     depends_on('py-setuptools', type=('build'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-cinemasci/package.py:38]: Missing dep on py-pyyaml

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-cinemasci/package.py (hunk 2 above)
Comment (EthanS94) [reply var/spack/repos/builtin/packages/py-cinemasci/package.py:38]: @adamjstewart I built without it. Why does it need py-pyyaml?

Comment (EthanS94) [reply var/spack/repos/builtin/packages/py-cinemasci/package.py:38]: You're right. Adding that.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**